test/wallet-http-test.js

```diff
@@ -188,6 +188,50 @@ describe('Wallet HTTP', function() {
       assert.ok(typeof path.change === 'boolean');
       assert.ok(typeof path.derivation === 'string');
     }
+
+    // cbAddress is a self-send
+    // so all output paths including change should be known
+    for (let i = 0; i < tx.outputs.length; i++) {
+      const path = tx.outputs[i].path;
+
+      assert.ok(typeof path.name === 'string');
```

Comment:
Any reason this test uses `assert.ok`? I thought hsd prefers `assert.strictEqual`

Reply:
For bools, it could go either `assert(..)` or `assert.ok(`. `assert.ok` preferred in tests.